motor_joints:
  - joint_name: drive_shaft_motor_joint
    node_id: 1
    min_velocity: # rpm
    max_velocity: # rpm
    ratio: # gearbox ratio * drive shaft ratio

servo_joints:
  - name: front_wheel_steer_servo_joint
    command_topic: /front_wheel_steer_controller/command
    min_position: # rads
    max_position: # rads
  - name: rear_wheel_steer_servo_joint
    command_topic: /rear_wheel_steer_controller/command
    min_position: # rads
    max_position: # rads

trapezoidal_steer_mechanism:
  offset_arm_length: 0.03  # [m]
  angle_beta: 0.083426738  # [rad] = 4.88 degrees
  long_base: 0.23  # [m]
  short_base: 0.225  # [m]
